R/paste.R

```diff
@@ -23,7 +25,7 @@
 #'
 #' dm_nycflights13() %>%
 #'   dm_paste(select = TRUE)
-dm_paste <- function(dm, select = FALSE, tab_width = 2) {
+dm_paste <- function(dm, select = FALSE, color = TRUE, tab_width = 2) {
```

Comment:
```suggestion
dm_paste <- function(dm, select = FALSE, tab_width = 2, color = TRUE) {
```

For better backward compatibility. Perhaps we can always generate colors and (if ever needed) provide `dm_remove_colors()` ?

Reply:
What would you think of leaving the color optional for `dm_paste()`, but anyway provide `dm_remove_colors()` (or maybe `dm_rm_colors()`?)